get a prod package

monsieurleberre opened the pull request.

monsieurleberre committed:
Upgrade packages and get Ci/Cd secrets from AWS (#37)

* Upgrade packages and get Ci/Cd secrets from AWS

* upgrade libraries and actions

* update README.md

* update README.md

* don't publish if build failed

* only publish on successful build

* only publish nuget from success test suite

dependabot[bot] committed:
chore(deps): bump Trakx.Utils.Testing from 0.3.13 to 0.6.7 (#59)

Bumps Trakx.Utils.Testing from 0.3.13 to 0.6.7.

---
updated-dependencies:
- dependency-name: Trakx.Utils.Testing
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.Extensions.Http.Polly from 6.0.4 to 6.0.6 (#50)

Bumps [Microsoft.Extensions.Http.Polly](https://github.com/dotnet/aspnetcore) from 6.0.4 to 6.0.6.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.4...v6.0.6)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Http.Polly
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils from 0.3.14 to 0.6.7 (#58)

Bumps Trakx.Utils from 0.3.14 to 0.6.7.

---
updated-dependencies:
- dependency-name: Trakx.Utils
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump GitHubActionsTestLogger from 2.0.0 to 2.0.1 (#60)

Bumps [GitHubActionsTestLogger](https://github.com/Tyrrrz/GitHubActionsTestLogger) from 2.0.0 to 2.0.1.
- [Release notes](https://github.com/Tyrrrz/GitHubActionsTestLogger/releases)
- [Changelog](https://github.com/Tyrrrz/GitHubActionsTestLogger/blob/master/Changelog.md)
- [Commits](https://github.com/Tyrrrz/GitHubActionsTestLogger/compare/2.0...2.0.1)

---
updated-dependencies:
- dependency-name: GitHubActionsTestLogger
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump NSwag.MSBuild from 13.16.0 to 13.16.1 (#61)

Bumps [NSwag.MSBuild](https://github.com/RicoSuter/NSwag) from 13.16.0 to 13.16.1.
- [Release notes](https://github.com/RicoSuter/NSwag/releases)
- [Changelog](https://github.com/RicoSuter/NSwag/blob/master/CHANGELOG.md)
- [Commits](https://github.com/RicoSuter/NSwag/compare/v13.16.0...v13.16.1)

---
updated-dependencies:
- dependency-name: NSwag.MSBuild
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.AspNetCore.Mvc.NewtonsoftJson (#51)

Bumps [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://github.com/dotnet/aspnetcore) from 6.0.4 to 6.0.6.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.4...v6.0.6)

---
updated-dependencies:
- dependency-name: Microsoft.AspNetCore.Mvc.NewtonsoftJson
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils.Testing from 0.6.7 to 0.6.9 (#62)

Bumps Trakx.Utils.Testing from 0.6.7 to 0.6.9.

---
updated-dependencies:
- dependency-name: Trakx.Utils.Testing
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.AspNetCore.Mvc.NewtonsoftJson (#64)

Bumps [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://github.com/dotnet/aspnetcore) from 6.0.6 to 6.0.7.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.6...v6.0.7)

---
updated-dependencies:
- dependency-name: Microsoft.AspNetCore.Mvc.NewtonsoftJson
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.Extensions.Http.Polly from 6.0.6 to 6.0.7 (#65)

Bumps [Microsoft.Extensions.Http.Polly](https://github.com/dotnet/aspnetcore) from 6.0.6 to 6.0.7.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.6...v6.0.7)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Http.Polly
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils from 0.6.7 to 0.6.10 (#66)

Bumps Trakx.Utils from 0.6.7 to 0.6.10.

---
updated-dependencies:
- dependency-name: Trakx.Utils
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump xunit from 2.4.1 to 2.4.2 (#68)

Bumps [xunit](https://github.com/xunit/xunit) from 2.4.1 to 2.4.2.
- [Release notes](https://github.com/xunit/xunit/releases)
- [Commits](https://github.com/xunit/xunit/compare/2.4.1...2.4.2)

---
updated-dependencies:
- dependency-name: xunit
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils.Testing from 0.6.9 to 0.6.10 (#67)

Bumps Trakx.Utils.Testing from 0.6.9 to 0.6.10.

---
updated-dependencies:
- dependency-name: Trakx.Utils.Testing
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.AspNetCore.Mvc.NewtonsoftJson (#69)

Bumps [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://github.com/dotnet/aspnetcore) from 6.0.7 to 6.0.8.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.7...v6.0.8)

---
updated-dependencies:
- dependency-name: Microsoft.AspNetCore.Mvc.NewtonsoftJson
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.Extensions.Http.Polly from 6.0.7 to 6.0.8 (#70)

Bumps [Microsoft.Extensions.Http.Polly](https://github.com/dotnet/aspnetcore) from 6.0.7 to 6.0.8.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.7...v6.0.8)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Http.Polly
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.NET.Test.Sdk from 17.2.0 to 17.3.0 (#71)

Bumps [Microsoft.NET.Test.Sdk](https://github.com/microsoft/vstest) from 17.2.0 to 17.3.0.
- [Release notes](https://github.com/microsoft/vstest/releases)
- [Commits](https://github.com/microsoft/vstest/compare/v17.2.0...v17.3.0)

---
updated-dependencies:
- dependency-name: Microsoft.NET.Test.Sdk
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils from 0.6.10 to 0.6.11 (#72)

Bumps Trakx.Utils from 0.6.10 to 0.6.11.

---
updated-dependencies:
- dependency-name: Trakx.Utils
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils.Testing from 0.6.10 to 0.6.11 (#73)

Bumps Trakx.Utils.Testing from 0.6.10 to 0.6.11.

---
updated-dependencies:
- dependency-name: Trakx.Utils.Testing
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils.Testing from 0.6.11 to 0.6.12 (#75)

Bumps Trakx.Utils.Testing from 0.6.11 to 0.6.12.

---
updated-dependencies:
- dependency-name: Trakx.Utils.Testing
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Trakx.Utils from 0.6.11 to 0.6.12 (#74)

Bumps Trakx.Utils from 0.6.11 to 0.6.12.

---
updated-dependencies:
- dependency-name: Trakx.Utils
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.NET.Test.Sdk from 17.3.0 to 17.3.1 (#76)

Bumps [Microsoft.NET.Test.Sdk](https://github.com/microsoft/vstest) from 17.3.0 to 17.3.1.
- [Release notes](https://github.com/microsoft/vstest/releases)
- [Commits](https://github.com/microsoft/vstest/compare/v17.3.0...v17.3.1)

---
updated-dependencies:
- dependency-name: Microsoft.NET.Test.Sdk
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Serilog from 2.11.0 to 2.12.0 (#77)

Bumps [Serilog](https://github.com/serilog/serilog) from 2.11.0 to 2.12.0.
- [Release notes](https://github.com/serilog/serilog/releases)
- [Changelog](https://github.com/serilog/serilog/blob/dev/CHANGES.md)
- [Commits](https://github.com/serilog/serilog/compare/v2.11.0...v2.12.0)

---
updated-dependencies:
- dependency-name: Serilog
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.Extensions.Http.Polly from 6.0.8 to 6.0.9 (#78)

Bumps [Microsoft.Extensions.Http.Polly](https://github.com/dotnet/aspnetcore) from 6.0.8 to 6.0.9.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.8...v6.0.9)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Http.Polly
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.AspNetCore.Mvc.NewtonsoftJson (#79)

Bumps [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://github.com/dotnet/aspnetcore) from 6.0.8 to 6.0.9.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.8...v6.0.9)

---
updated-dependencies:
- dependency-name: Microsoft.AspNetCore.Mvc.NewtonsoftJson
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump NSwag.MSBuild from 13.16.1 to 13.17.0 (#80)

Bumps [NSwag.MSBuild](https://github.com/RicoSuter/NSwag) from 13.16.1 to 13.17.0.
- [Release notes](https://github.com/RicoSuter/NSwag/releases)
- [Changelog](https://github.com/RicoSuter/NSwag/blob/master/CHANGELOG.md)
- [Commits](https://github.com/RicoSuter/NSwag/compare/v13.16.1...v13.17.0)

---
updated-dependencies:
- dependency-name: NSwag.MSBuild
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.NET.Test.Sdk from 17.3.1 to 17.3.2 (#81)

Bumps [Microsoft.NET.Test.Sdk](https://github.com/microsoft/vstest) from 17.3.1 to 17.3.2.
- [Release notes](https://github.com/microsoft/vstest/releases)
- [Commits](https://github.com/microsoft/vstest/compare/v17.3.1...v17.3.2)

---
updated-dependencies:
- dependency-name: Microsoft.NET.Test.Sdk
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.AspNetCore.Mvc.NewtonsoftJson (#82)

Bumps [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://github.com/dotnet/aspnetcore) from 6.0.9 to 6.0.10.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.9...v6.0.10)

---
updated-dependencies:
- dependency-name: Microsoft.AspNetCore.Mvc.NewtonsoftJson
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.Extensions.DependencyInjection (#83)

Bumps [Microsoft.Extensions.DependencyInjection](https://github.com/dotnet/runtime) from 6.0.0 to 6.0.1.
- [Release notes](https://github.com/dotnet/runtime/releases)
- [Commits](https://github.com/dotnet/runtime/compare/v6.0.0...v6.0.1)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.DependencyInjection
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump Microsoft.Extensions.Http.Polly from 6.0.9 to 6.0.10 (#84)

Bumps [Microsoft.Extensions.Http.Polly](https://github.com/dotnet/aspnetcore) from 6.0.9 to 6.0.10.
- [Release notes](https://github.com/dotnet/aspnetcore/releases)
- [Changelog](https://github.com/dotnet/aspnetcore/blob/main/docs/ReleasePlanning.md)
- [Commits](https://github.com/dotnet/aspnetcore/compare/v6.0.9...v6.0.10)

---
updated-dependencies:
- dependency-name: Microsoft.Extensions.Http.Polly
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump FluentAssertions from 6.7.0 to 6.8.0 (#85)

Bumps [FluentAssertions](https://github.com/fluentassertions/fluentassertions) from 6.7.0 to 6.8.0.
- [Release notes](https://github.com/fluentassertions/fluentassertions/releases)
- [Changelog](https://github.com/fluentassertions/fluentassertions/blob/develop/AcceptApiChanges.ps1)
- [Commits](https://github.com/fluentassertions/fluentassertions/compare/6.7.0...6.8.0)

---
updated-dependencies:
- dependency-name: FluentAssertions
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

monsieurleberre committed:
remove exchanges that we dont trust from supported exchanges (#86)

monsieurleberre committed:
Merge branch 'master' into dev

monsieurleberre committed:
remove old projects that make the build fail 🤒💩

monsieurleberre committed:
Merge branch 'master' into dev

monsieurleberre committed:
Change case of new exchanges, add a test to list them as they are getting serialised in Enum Values

monsieurleberre committed:
Merge remote-tracking branch 'origin/dev' into dev

dependabot[bot] committed:
chore(deps): bump coverlet.collector from 3.1.2 to 3.2.0 (#90)

Bumps [coverlet.collector](https://github.com/coverlet-coverage/coverlet) from 3.1.2 to 3.2.0.
- [Release notes](https://github.com/coverlet-coverage/coverlet/releases)
- [Commits](https://github.com/coverlet-coverage/coverlet/commits/v3.2.0)

---
updated-dependencies:
- dependency-name: coverlet.collector
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
chore(deps): bump coverlet.msbuild from 3.1.2 to 3.2.0 (#91)

Bumps [coverlet.msbuild](https://github.com/coverlet-coverage/coverlet) from 3.1.2 to 3.2.0.
- [Release notes](https://github.com/coverlet-coverage/coverlet/releases)
- [Commits](https://github.com/coverlet-coverage/coverlet/commits/v3.2.0)

---
updated-dependencies:
- dependency-name: coverlet.msbuild
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: monsieurleberre <monsieurleberre@users.noreply.github.com>

monsieurleberre committed:
Trying to change the minimum to fix the issue longer term (#111)

joaofbvieira commented on .github/workflows/delete.packages.yml:
```diff
@@ -0,0 +1,27 @@
+    steps:
+    - name: Delete packages
+      id: deletePackages
+      uses: actions/delete-package-versions@v3
```
Not a blocker for this PR, but could use `actions/delete-package-versions@v4` instead. There is an example in common repository.

monsieurleberre replied:
thanks, dependabot's PR just merged 🥳

joaofbvieira replied:
@monsieurleberre in order to use the new version of "delete-package-versions", it is required to add an extra parameter `package-type: 'nuget'`. This was already done in [common](https://github.com/trakx/common) repository.
<img width="711" alt="image" src="https://user-images.githubusercontent.com/6867927/216935684-67f003c6-62f2-4c18-9653-b32e1a5c0b10.png">

monsieurleberre replied:
OK Sorry I didn't see that, will do the change now! 🙏

monsieurleberre replied:
https://github.com/trakx/shrimpy-api-client/pull/114

joaofbvieira commented on src/Common.Projects.props:
```diff
@@ -1,7 +1,7 @@
     <PropertyGroup>
         <Company>Trakx</Company>
-        <Copyright>Copyright © 2021 Trakx</Copyright>
+        <Copyright>Copyright © 2022 Trakx</Copyright>
```
Not a blocker for now, but now it is 2023.

monsieurleberre replied:
thanks I opened #113 for it :)

joaofbvieira reviewed (dismissed):
LGTM. Added some suggestion, but are not a blocker for this PR.
Maybe need to solve merge conflicts before merge the PR.

monsieurleberre committed:
Merge branch 'master' into dev

dependabot[bot] committed:
chore(deps): bump actions/delete-package-versions from 3 to 4 (#107)

Bumps [actions/delete-package-versions](https://github.com/actions/delete-package-versions) from 3 to 4.
- [Release notes](https://github.com/actions/delete-package-versions/releases)
- [Commits](https://github.com/actions/delete-package-versions/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/delete-package-versions
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: monsieurleberre <monsieurleberre@users.noreply.github.com>

monsieurleberre committed:
update copyright year (#113)

* update copyright year

* fix copyright year

monsieurleberre committed:
fix delete package to have type param (#114)

joaofbvieira reviewed (approved):
LGTM xD
